step-certificates/templates/ca.yaml

```diff
@@ -79,31 +114,84 @@ spec:
           - name: secrets
             mountPath: /home/step/secrets
             readOnly: true
+          {{- if or (and .Values.existingSecrets.enabled .Values.existingSecrets.ca)  (or (and .Values.bootstrap.enabled .Values.bootstrap.secrets) (and .Values.inject.enabled (not (eq .Values.inject.secrets.ca_password "")))) }}
           - name: ca-password
             mountPath: /home/step/secrets/passwords
             readOnly: true
+          {{- end }}
           {{- if .Values.ca.db.enabled }}
           - name: database
             mountPath: /home/step/db
             readOnly: false
           {{- end }}
+          {{- if or (and .Values.existingSecrets.enabled .Values.existingSecrets.issuer)  (and .Values.inject.enabled (and .Values.inject.secrets.certificate_issuer.enabled (not (eq .Values.inject.secrets.certificate_issuer.password "")))) }}
+          - name: certificate-issuer
+            mountPath: /home/step/secrets/certificate-issuer
+            readOnly: true
+          {{- end }}
+          {{- if or (and .Values.existingSecrets.enabled .Values.existingSecrets.sshHostCa)  (and .Values.inject.enabled (and .Values.inject.secrets.ssh.enabled (not (eq .Values.inject.secrets.ssh.host_ca_password "")))) }}
+          - name: ssh-host-ca-password
+            mountPath: /home/step/secrets/ssh-host-ca
+            readOnly: true
+          {{- end }}
+          {{- if or (and .Values.existingSecrets.enabled .Values.existingSecrets.sshUserCa) (and .Values.inject.enabled (and .Values.inject.secrets.ssh.enabled (not (eq .Values.inject.secrets.ssh.user_ca_password "")))) }}
+          - name: ssh-user-ca-password
+            mountPath: /home/step/secrets/ssh-user-ca
+            readOnly: true
+          {{- end }}
       volumes:
+      - configMap:
+          name: custom-trusted-ca
+          optional: true
+        name: custom-trusted-ca
```

Comment:
This is the second and last change that I've noticed in this file. This is the one I referenced before and should be added adding and property that allows to extend volumes as much as you want.